src/Statamic.php

```diff
@@ -374,4 +375,27 @@ public static function trans($key, $replace = [], $locale = null)
 
         return $line;
     }
+
+    private static function createVersionedAssetPath($name, $path, $type)
+    {
+        $ttl = now()->addDay();
+
+        return Cache::remember("statamic-{$type}-{$name}", $ttl, function () use ($path, $type) {
```

Comment:
This can probably just be rememberForever. If you're needing it to be busted, you're not going to want to wait a day anyway.

Reply:
I agree. Changed.